src/garage/np/algos/off_policy_rl_algorithm.py

```diff
@@ -169,3 +178,63 @@ def train_once(self, itr, paths):
 
         """
         raise NotImplementedError
+
+    def _obtain_evaluation_samples(self, batch_size=1000):
+        r"""Sample the policy for 10 trajectories and return average values.
+
+        Args:
+            batch_size (int): Number of environment steps in one batch.
+
+        Returns:
+            dict: Evaluation trajectories, representing
+                the best current performance of the algorithm, with keys:
```

Comment:
hmm this is wrapped strangely -- isn't there room on the previous line for `the base current performance`?